### PR TITLE
People

### PR DESCRIPTION
**Pull request by FredPeal**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -27,6 +27,7 @@ import {
   FileSystem,
   Topics,
   SystemModules,
+  Peoples,
```
Comment (RivierGrullon) [inline src/index.ts:30]: Change: people is already plural, remove the `s`

File: src/index.ts [hunk 2]
```diff
@@ -105,9 +106,10 @@ export default class KanvasCore {
   public filesystem: FileSystem;
   public topics: Topics;
   public systemModules: SystemModules;
-  
+
   public companies: Companies;
   public companiesBranches: CompaniesBranches;
+  public peoples: Peoples;
```
Comment (RivierGrullon) [inline src/index.ts:112]: Same

File: src/index.ts [hunk 3]
```diff
@@ -136,6 +138,7 @@ export default class KanvasCore {
     this.systemModules = new SystemModules(this.client);
     this.companies = new Companies(this.client);
     this.companiesBranches = new CompaniesBranches(this.client);
+    this.peoples = new Peoples(this.client);
```
Comment (RivierGrullon) [inline src/index.ts:141]: same here

File: src/modules/index.ts [hunk 4]
```diff
@@ -16,3 +16,4 @@ export * from './topics';
 export * from './system-modules';
 export * from './companies';
 export * from './companies-branches';
+export * from './peoples';
```
Comment (RivierGrullon) [inline src/modules/index.ts:19]: and here

File: src/modules/peoples/index.ts [hunk 5]
```diff
@@ -0,0 +1,77 @@
+import { ClientType } from '../../index';
```
Comment (RivierGrullon) [inline src/modules/peoples/index.ts:1]: change: the name of the folder here too

File: src/queries/index.ts [hunk 6]
```diff
@@ -14,3 +14,4 @@ export * from './topics.query';
 export * from './system-module.query';
 export * from './companies-branches.query';
 export * from './companies.query';
+export * from './peoples.query';
```
Comment (RivierGrullon) [inline src/queries/index.ts:17]: same here

File: src/queries/peoples.query.ts [hunk 7]
```diff
@@ -0,0 +1,59 @@
+import { gql } from '@apollo/client/core';
+
+export const PEOPLES_QUERY = gql`
+  query peoples(
+    $where: WhereCondition
```
Comment (RivierGrullon) [inline src/queries/peoples.query.ts:5]: change: the name of the file here and the constants too

File: src/types/leads.ts [hunk 8]
```diff
@@ -2,27 +2,29 @@
...
-  firstname?: string;
-  lastname?: string;
-  contacts: Contact[];
-  address?: Address[];
-  custom_fields?: CustomFieldInput[];
-}
+import {PeopleInterface} from './index';
+// interface Contact {
+//   value: string;
+//   contacts_types_id?: number;
```
Comment (RivierGrullon) [inline src/types/leads.ts:8]: Question: why is this commented on? if we are not using it we can remove these comments

**Review by RivierGrullon — CHANGES_REQUESTED** [review]

File: src/types/leads.ts (hunk 8 above)
Comment (FredPeal) [reply src/types/leads.ts:8]: Because I want the confirmation from @jerlyrosa and @lizandro-mc

**Review by RivierGrullon — APPROVED** [review]

**Pull request merged**